package.json

```
@@ -26,18 +26,22 @@
   ],
   "repository": "https://github.com/salesforce/wire-service-jest-util",
   "peerDependencies": {
-    "@lwc/engine": "^1.5.0"
+    "@lwc/engine-dom": "^2.0.0"
```

What about using a peer dependency against the umbrella package instead of `@lwc/engine-dom`? IMO, we should avoid asking developers to import each piece of the LWC framework independently.

```suggestion
    "lwc": "^2.0.0"
```

---

I brought up this concern for https://github.com/salesforce/lwc-test/pull/110#discussion_r562441539 but we ended up not doing so after some discussion. @jodarove do you remember what the reasoning was?